index.html

```diff
@@ -14,5 +14,38 @@
     <title>BookWise || Read, Discover, and Share</title>
   </head>
   <body>
+    <nav class="header">
+      <div class="header__logo-container">
+        <img
+          src="./src/img/logo.png"
+          alt="logo"
```

Comment:
logo is not a good enough alt text here

https://www.a11yproject.com/posts/alt-text/#a-few-tips-on-how-and-when-to-use-the-alt-attribute%3A

Reply:
Changed the alt to "bookwise"